examples/quantumgames/example_extended_nonlocal_games.py

```diff
@@ -0,0 +1,708 @@
+"""Extended nonlocal games
+==========================
+
+In this tutorial, we will define the concept of an *extended nonlocal game*.
+Extended nonlocal games are a more general abstraction of nonlocal games
+wherein the referee, who previously only provided questions and answers to the
+players, now share a state with the players and is able to perform a
+measurement on that shared state.
+"""
+# sphinx_gallery_thumbnail_path = 'figures/extended_nonlocal_game.svg'
```

Comment:
same as my other comment, this line appears in the build. If I look at the examples in the sphinx-gallery documentation, they are not present in the docs build.